src/modules/crypto/encryptedDataDeserialization.ts

```diff
@@ -128,7 +128,7 @@ const deserializeSymmetricCipherConfig = (
 
     const mode = extractNextEncryptedDataStringComponent(encryptedDataString.substring(cursor));
     cursor += mode.cursorAfter;
-    if (mode.component !== 'cbchmac') {
+    if (mode.component !== 'cbchmac' && mode.component !== 'cbchmac64') {
```

Comment:
I don't like this form but that's just me

`if (!(mode.component === 'cbchmac' || mode.component === 'cbchmac64'))`

Reply:
same stuff